app/navigation/Navigation.scala

```diff
@@ -73,7 +73,8 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
     AppealStartPage -> ((_, _, _) => routes.AppealStartController.onPageLoad()),
     OtherRelevantInformationPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadForUploadEvidence(NormalMode)),
     CancelVATRegistrationPage -> ((answer, _, extraData) => routingForCancelVATRegistrationPage(answer, extraData)),
-    OtherPenaltiesForPeriodPage -> ((_, _, _) => routes.AppealStartController.onPageLoad())
+    OtherPenaltiesForPeriodPage -> ((_, _, _) => routes.AppealStartController.onPageLoad()),
+    UploadCompletePage -> ((_, _, _) => routes.OtherReasonController.onPageLoadForAnotherFileUpload(NormalMode))
```

Comment:
We need custom routing for this: 
if the user clicks yes - then we go to onPageLoadForAnotherFileUpload
if the user clicks no - then we go to the next page (either why appeal is late, or CYA)